usaspending_api/settings.py

```diff
@@ -48,7 +48,7 @@
     USASPENDING_AWS_REGION = os.environ.get("USASPENDING_AWS_REGION")
 
 # AWS locations for CSV files
-CSV_LOCAL_PATH = str(BASE_DIR / "csv_downloads")
+CSV_LOCAL_PATH = str(BASE_DIR / "csv_downloads") + "/"
```

Comment:
Unable to comment on these lines in settings.py which also should be directories:
Line 248, 249,